src/PuliPluginImpl.php

```diff
@@ -200,6 +251,13 @@ public function postInstall()
 
     private function initialize()
     {
+        if (!file_exists($this->autoloadFile)) {
+            $filesystem = new Filesystem();
+            // Avoid problems if using the runner before autoload.php has been
+            // generated
```

Comment:
Yes. Puli CLI, while doing the version check, tries to include it, leading to a warning message.